aws-samples/aws-analytics-reference-architecture
# feat(refarch): Move redshift SQL bootstrapping to core/db-schema-manager (flyway)

Move redshift bootstraping to core/db-schema-manager construct.

requires #217

### testing procedure

1. pull the branch combining #217 and #218 
  ```sh
  git checkout refarch/full
  ```

2. Build core
  ```
  cd core && npx projen build
  ```

3. install new deps in refarch
   ```
    cd ../refarch/aws-native && pip install --no-cache-dir -r requirements.txt
   ```

4. deploy
  ```
  cdk deploy
  ```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
